### ISSUE TITLE
Hotmail/Outlook as the Transport is broken

### ISSUE THREAD

**Issue by jeremykentbgross** [nodemailer/nodemailer#1685]
Using hotmail credentials no longer works and generates the following error:
`535 5.7.139 Authentication unsuccessful, basic authentication is disabled.`

It seems that this is because Microsoft has changed their login method to OAuth2, as documented here:
https://answers.microsoft.com/en-us/outlook_com/forum/all/error-535-57139-authentication-unsuccessful-basic/6a18c08d-6163-4a44-8e09-33bdecfb1d0f

It looks like there is an OAuth2 folder in the nodemailer codebase, but I am not familiar enough with the internals to know what all needs to change in the code for the Hotmail connections to use it, so I figured filing a report was best.

**Comment by andris9** [comment]
See this comment: https://github.com/nodemailer/nodemailer/issues/1485#issuecomment-2396515467

**Comment by jeremykentbgross** [comment]
@andris9 Thank you for the reply.  I noticed in the thread you linked, a different comment you made that also seems relevant:

(https://github.com/nodemailer/nodemailer/issues/1485#issuecomment-1320725372)

> For Gmail accounts Nodemailer can renew the access token itself. For MS365 accounts, it can’t and you have to provision that token yourself.

At the time I implemented my current solution, the documentation at https://nodemailer.com/ had said something about GMail tending not to work because of some security issue (and indeed I seem to recall testing that it rejected sending my mail), although why that was I don't recall and I don't see that on the page now.

I had created a specific Hotmail account for the sole purpose of sending my generated emails.  Do I understand now from your comment that I would be best off by switch to using a GMail account for this purpose instead?

It seems that in any case I have to learn about OAuth2 and how to generate these access tokens, but if these need constant renewal which isn't supported by the MS360 accounts, I might as well switch to GMail, or so your comment would seem to indicate.

Thanks for your comment and work on the project.
